WalletWasabi/Bases/NotifyPropertyChangedBase.cs

```diff
@@ -27,6 +27,18 @@ protected bool RaiseAndSetIfChanged<T>(ref T field, T value, [CallerMemberName]
 			return true;
 		}
 
+		protected bool RaiseAndSetIfRefChanged<T>(ref T field, ref T value, [CallerMemberName] string propertyName = null)
+		{
+			if (ReferenceEquals(field, value))
```

Comment:
What is the difference between this and the old equality compared?

Reply:
It compares reference, not equality.